tools/slicec-cs/src/generators/exception_generator.rs

```diff
@@ -1,41 +1,53 @@
 // Copyright (c) ZeroC, Inc.
 
-use crate::builders::{AttributeBuilder, Builder, CommentBuilder, ContainerBuilder, FunctionBuilder, FunctionType};
+use crate::builders::{
```

Comment:
Now a copy-paste of class_generator, with a few updates like:
 - Class => Exception
 - no compact type ID.